bundles/org.eclipse.e4.ui.workbench/src/org/eclipse/e4/ui/activitysupport/IActivityManagerProxy.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Advantest Europe GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ * 				Raghunandana Murthappa
+ *******************************************************************************/
+package org.eclipse.e4.ui.activitysupport;
+
+/**
+ * A bridge between org.eclipse.ui.workbench and
+ * org.eclipse.e4.ui.workbench.renderers.swt.
+ *
+ * Service for this interface is bound to Platform.class bundle at
+ * Workbench.class. We cannot depend on org.eclipse.ui.workbench from
+ * org.eclipse.e4.ui.workbench.renderers.swt
+ *
+ * @since 1.17
+ */
+public interface IActivityManagerProxy {
```

Comment:
Can we not simply name it

```suggestion
public interface IActivityManager {
```

Also I don't think we need to mention Platform (UI) and workbench here, in general we just have a new E4 interface and the "old" E3 implements that interface